semana15/projeto-sisBancario/src/index.ts

```diff
@@ -0,0 +1,85 @@
+import express, { Express, Request, Response } from "express";
+import cors from "cors";
+import { AddressInfo } from "net";
+import { user } from './data';
+import { accounts } from "./accounts";
+
+const app: Express = express()
+
+app.use(express.json())
+app.use(cors())
+
+
+app.post("/user/resgister", (res: Response, req: Request) => {
+    try {
+        const { name, CPF, birthDate } = req.body
+        const limitBirthDate = new Date('2003-09-20')
```

Comment:
Esta data vai mudar dependendo do dia em que o sistema for usado, correto? 
Portanto, o ideal é não deixar um valor fixo, mas sim usar o `Date.now()` para buscar a data atual e a partir daí realizar operações para encontrar a data limite de nascimento ou até mesmo já verificar a idade em anos do usuário.